tests/s25Main/UI/testSmartBitmap.cpp

```diff
@@ -278,4 +351,56 @@ BOOST_AUTO_TEST_CASE(MultiPlayerBitmap)
     }
 }
 
+BOOST_AUTO_TEST_CASE(DrawPercent)
+{
+    // Make sure that test passes across different image heights
+    std::vector<Extent> testCases = {
+      {rttr::test::randomValue<unsigned int>(10, 50), 50},
+      {rttr::test::randomValue<unsigned int>(10, 50), 35},
```

Comment:
Is my tooling acting up or does clangformat format such initializers with different indent?

Probably also should've used std::to_array instead...

Reply:
There are "full indents" and "continuation indents". I.e. when a new block starts or when the previous line/statement is continued after being broken up due to max. line length